include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -2565,8 +2565,7 @@ inline unsigned int
 DoFAccessor<0, DoFHandlerType<1, spacedim>, level_dof_access>::
   n_active_fe_indices() const
 {
-  Assert(false, ExcNotImplemented());
-  return numbers::invalid_unsigned_int;
+  return 1;
```

Comment:
This (and the other function you have below) are correct if `DoFHandlerType` is `::DoFHandler`, but not if it is a `hp::DoFHandler`. Do you know whether there are specializations? You might want to take a look at how this function is implemented in the case of `dim>1`.

Alternatively, if that is too complicated, you could add an `Assert` checking that `std::same_type<DoFHandlerType,DoFHandler<1,spacedim>>::value == true`, saying that the other cases are not implemented.

Reply:
@bangerth: thanks! I have chosen the approach with the asserts. 

I have tried to implement the `hp::DoFHandler`-version, however, was not able to find an equivalent of `this->present_index` for `DoFAccessor<0,..>`. There is only `global_vertex_index`, which is obviously not equivalent...